security/webauthn/src/test/java/OpenshiftWebAuthnIT.java

```diff
@@ -0,0 +1,15 @@
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@OpenShiftScenario
+public class OpenshiftWebAuthnIT extends AbstractWebAuthnTest {
```

Comment:
It needs to be `OpenShiftWebAuthIT` for there is Maven exclude pattern and with lower case, this test is run in a bare metal.

Reply:
Good catch!, changing class name to `OpenShiftWebAuthIT`